core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/CoherenceClusterSystemResource.json

```diff
@@ -29,22 +31,26 @@
         },
         "CoherenceResource": {
             "wlst_type": "${CoherenceResource:CoherenceClusterResource}",
+            "online_bean": "weblogic.coherence.descriptor.wl.WeblogicCoherenceBean",
             "version" : "[12.1.2,)",
             "child_folders_type": "single",
             "default_name_value": "%COHERENCECLUSTER%",
             "folders": {
                 "CoherenceAddressProvider": {
                     "wlst_type": "CoherenceAddressProvider${:s}",
+                    "online_bean": "weblogic.coherence.descriptor.wl.CoherenceAddressProvidersBean",
                     "version" : "[12.2.1.1,)",
                     "child_folders_type": "multiple",
                     "flattened_folder_data": {
-                        "wlst_type": "CoherenceAddressProviders",
+                        "wlst_type": "CoherenceAddressProvider",
```

Comment:
Are we sure about this change?  Changing the `wlst_type` has big implications so I just need to understand this change and why it is needed.

Reply:
@ddsharpe Can someone please verify this change to the `wlst_type`?  I suspect this could be an offline vs. online this where the right change is to accommodate both...

Reply:
Thanks. Reverted.